ambry-store/src/main/java/com/github/ambry/store/FileStore.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2024 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import java.nio.channels.FileChannel;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class FileStore {
```

Comment:
Check if this can be made "package-private".
Thats is a forcing function to keep classes isolated and organised.

Reply:
done